### PR TITLE
TST: increased test coverage to 98 percent

### LINKED ISSUE #55: increase test coverage
This plugin is currently around 90% covered with unit tests - it looks like there is some important functionality that should still be tested based on the coverage report.

### PR DESCRIPTION
**Pull request by nbokulich**
fixes #55

### CODE REVIEW

File: q2_sample_classifier/tests/test_classifier.py [hunk 1]
```diff
@@ -127,7 +290,7 @@ def test_classifiers(self):
                            'LinearSVC', 'SVC', 'KNeighborsClassifier']:
             tmpd = join(self.temp_dir.name, classifier)
             mkdir(tmpd)
-            estimator, pd, pt = _set_parameters_and_estimator(
+            estimator, pad, pt = _set_parameters_and_estimator(
```
[the remark this answers is not in the document]
Comment (thermokarst) [reply q2_sample_classifier/tests/test_classifier.py]: :nbokulich:

[the remark this answers is not in the document]
Comment (nbokulich) [reply q2_sample_classifier/tests/test_classifier.py]: :man_facepalming:

File: q2_sample_classifier/classify.py [hunk 2]
```diff
@@ -158,7 +158,7 @@ def detect_outliers(table: pd.DataFrame,
 
     # if opting to train on a subset, choose subset that fits criteria
     if subset_category and subset_value:
-        y_train = metadata[metadata[subset_category] == subset_value]
+        y_train = sample_md[sample_md[subset_category] == subset_value]
```
[the remark this answers is not in the document]
Comment (nbokulich) [reply q2_sample_classifier/classify.py]: no — it just cropped up during the most recent changes, though unfortunately I can't remember how I discovered it.

**Comment by nbokulich** [comment]
Awesome, thanks @thermokarst ! and thanks for pointing out the issue with super.

**Pull request merged**